package.json

```diff
@@ -54,17 +48,16 @@
     "@typescript-eslint/parser": "^4.21.0",
     "babel-loader": "^8.2.2",
     "editorconfig-checker": "^4.0.2",
-    "eslint-config-xo": "^0.38.0",
-    "eslint-config-xo-react": "^0.25.0",
-    "eslint-config-xo-typescript": "^0.38.0",
-    "eslint-plugin-react": "^7.23.1",
+    "eslint": "^7.32.0",
+    "eslint-config-prettier": "^8.3.0",
```

Comment:
As far as I see this is not actually extended in eslintrc. Does `plugin:prettier/recommended` automatically add this? And if so, do we need to still keep it as an explicit dependency?